hologram_dashboard/extension/src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, throwError } from 'rxjs';
+import { catchError, retry } from 'rxjs/operators';
+
+@Injectable()
+export class RequestHandler {
+  url: string
+  constructor(private http: HttpClient) { 
+    this.url = "http://localhost:4200/"
+  }
+
+  getDashboard(system: string) {
+    let html = this.url.concat(system);
+    return this.http.get(html);
+  }
+}
+
+@Component({
+  selector: 'app-dashboard',
+  templateUrl: './dashboard.component.html',
+  styleUrls: ['./dashboard.component.css'],
+  providers: [ RequestHandler ]
+})
+export class DashboardComponent implements OnInit {
+  chipper_dashboard: string;
+  chipper_gdpr_dashboard: string;
+
+  constructor(private requestHandler: RequestHandler) { 
+    this.chipper_dashboard = "";
+    this.chipper_gdpr_dashboard = "";
+  }
+
+  ngOnInit(): void {
+  }
+
+  onClick(system: string) {
+    this.requestHandler.getDashboard(system).toPromise()
+      .then((data) => {this.chipper_dashboard = data["message"]})
```

Comment:
ooc, do you still request the server to send back the dashboard page? I think we should communicate with server using json object, and client will construct the dashboard?

Reply:
No this is only a test to see whether the connection with backend worked.